src/appShell/App/PortalFooter.tsx

```diff
@@ -30,10 +30,10 @@ export default class PortalFooter extends React.Component<{}, {}> {
                                     <h3>HELP</h3>
                                     <ul>
                                         <If condition={AppConfig.serverConfig.skin_show_tutorials_tab !== false}>
-                                            <li><a href="s/tutorials">Tutorials</a></li>
+                                            <li><a href="/tutorials">Tutorials</a></li>
```

Comment:
@alisman 	should be `tutorials` such that it respects subfolders e.g. www.cbioportal.org/beta/